src/pages/EditMonitor.vue

```diff
@@ -664,6 +668,58 @@
                                 </div>
                             </template>
 
+                            <template v-if="monitor.type === 'system-service'">
+                                <div class="my-3">
+                                    <label for="system-service-name" class="form-label">{{ $t("Service Name") }}</label>
+                                    <input id="system-service-name" v-model="monitor.system_service_name" type="text" class="form-control" required placeholder="nginx">
+                                    <div class="form-text">
+                                        <template v-if="$root.info.runtime.platform === 'linux'">
+                                            {{ $t("systemServiceDescriptionLinux") }} {{ monitor.system_service_name || 'nginx' }} {{ $t("systemServiceActiveText") }}
+                                        </template>
+                                        <template v-else-if="$root.info.runtime.platform === 'win32'">
+                                            {{ $t("systemServiceDescriptionWindows") }} {{ monitor.system_service_name || 'nginx' }} {{ $t("systemServiceActiveText") }}
+                                        </template>
+                                        <template v-else>
+                                            {{ $t("systemServiceDescription") }} {{ monitor.system_service_name || 'nginx' }} {{ $t("systemServiceActiveText") }}
+                                        </template>
+
+                                        <div class="mt-2">
+                                            <details v-if="$root.info.runtime.platform === 'linux'" class="mb-2">
+                                                <summary
+                                                    class="btn btn-outline-primary btn-sm w-100 text-start"
+                                                    style="box-shadow: none !important;
+
+                                                        --bs-btn-focus-box-shadow: none;"
+                                                >
+                                                    <i class="fab fa-linux me-1"></i>
+                                                    {{ $t("systemServiceLinuxLabel") }}
+                                                </summary>
+                                                <div class="p-2 ps-4">
```

Comment:
this is very inconsistent with the rest of the UI, please change this to be a regular helptext via `i18n-t`.

Docs: https://vue-i18n.intlify.dev/guide/advanced/component